### ISSUE TITLE
Implement Hero Section of Homepage

### ISSUE THREAD

**Issue by akshit-g** [GDSC-UPES/GDSC-UPES-Website#2]
Create and implement the hero section of the homepage using the provided UI/UX assets. The hero section is the first visible area of the homepage

Responsive Design: Ensure the hero section is responsive and displays well on various devices and screen sizes (desktops, tablets, mobile phones). Implement responsive design techniques to adapt the layout and content accordingly. Use media query to create the responsive design.

![image](https://github.com/GDSC-UPES/GDSC-UPES-Website/assets/90208584/5c53979c-860c-4901-a9e4-fcd52f223bbc)

**Comment by ansisme** [comment]
hey, is there a figma link to the ui/ux design of the whole website?

**Comment by rohin079** [comment]
> hey, is there a figma link to the ui/ux design of the whole website?

Yes @ansisme Here's the link: https://www.figma.com/file/9fJtrEvFUEkbuVijMA9d31/GDSC-UPES-(Copy)?type=design&node-id=8-198&mode=design

**Comment by ansisme** [comment]
Thanks, would like to take up the issue!

**Comment by akshit-g** [comment]
> Thanks, would like to take up the issue!

Sure @ansisme

**Comment by ansisme** [comment]
Thanks !

**Comment by akshit-g** [comment]
> Can I work on this?
> 

Sure @Supragya1

**Comment by ansisme** [comment]
error:
![image](https://github.com/GDSC-UPES/GDSC-UPES-Website/assets/91877814/ec4ead57-28e3-4b9f-a616-b376cc1267f7)
solution:
![image](https://github.com/GDSC-UPES/GDSC-UPES-Website/assets/91877814/4bbb6564-675e-413a-b264-27e453d54c16)

there is a restriction in importing the images outside of src folder, so would it be okay if I change the project structure and move the assets folder inside the src/static folder? it's working this way.

**Comment by rohin079** [comment]
@ansisme Instead of changing the folder structure you can copy the required images that you are using from assets folder and paste them in static and import them from static itself, assets folder is just for reference, it'll be removed after project has been completed

**Comment by ansisme** [comment]
Ok sure, thanks.

**Comment by ansisme** [comment]
please review the PR, thanks.

**Comment by rohin079** [comment]
This issue has been completed by @ansisme
